modules/va_forms/app/workers/va_forms/form_builder.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'va_forms/regex_helper'
+
+module VAForms
+  class FormBuilder
+    include Sidekiq::Worker
+    include SentryLogging
+    FORM_BASE_URL = 'https://www.va.gov'
+
+    def perform(form)
+      build_and_save_form(form)
+    rescue => e
+      message = "#{self.class.name} failed to import #{form['fieldVaFormNumber']} into forms database"
+      Rails.logger.error(message, e)
+      log_message_to_sentry(message, :error, body: e.message)
+    end
```

Comment:
Since Every exception is caught and not re-raised, sidekiq will always consider this job run successful and no retires will be attempted.  Not sure, but perhaps at least a few retires should be considered ?

Reply:
Agreed. If I re-raise in the `rescue` block, will that only re-raise `max_retries` (6) number of times?

Reply:
Yes, that is correct.

Reply:
I added a `raise` to the `rescue` block and set `retries` to 6. Should preclude any flaky server responses.